pipeline/src/lib.rs

```diff
@@ -97,8 +97,8 @@ pub fn serde_data_to_query_callback<T: FieldElement>(
     move |query: &str| -> Result<Option<T>, String> {
         let (id, data) = parse_query(query)?;
         match id {
-            "DataIdentifier" => {
-                let [index, cb_channel] = data[..] else {
```

Comment:
Was this a bug or did you reverse it in other places as well?

Reply:
this was like this here, but the other way around in the prover functions. I changed everything to (channel, idx)